.github/ISSUE_TEMPLATE/api_bug.md

```diff
@@ -0,0 +1,117 @@
+| Content-Type      | application/json              |
+| Transfer-Encoding | chunked                       |
+| Connection        | keep-alive                    |
+| X-Powered-By      | PHP/7.1.15                    |
+| Cache-Control     | no-cache, private             |
+| Date              | Sun, 25 Nov 2018 21:37:17 GMT |
+
+### Body
+
+```
+true
+```
+
+## Logs
+
+*Please attach the content of the following files to your bugs report :*
+ - *laravel-{date}.log (this file will contains all information regarding to error in code)*
+
+*Those files can be found in the following directory : `/var/www/seat/storage/logs/`*
+*For detailed instructions about how to retrieve them on either blade or Docker installation, please refer to the [documentation](https://eveseat.github.io/docs/troubleshooting/#checking-log-files)*
+
+*In case you're not providing the complete file, please surround the content with three grave accent before and after the content (```)*
+
+*Furthermore, ensure you're providing the full exception stack and not only the first or last line. Lines regarding an exception are numbered from 1 to infinite and preceded by two lines like :*
+```
+Some\Weired\Thing\StuffException in /path/to/the/file.php:14
+Stack trace:
+```
+
+*Here is a sample of Exception with the attached trace.*
+```
+[2018-11-18 14:41:36] production.ERROR: Class 'PHPExcel_Shared_Font' not found {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'PHPExcel_Shared_Font' not found at /var/www/seat/config/excel.php:182)
+[stacktrace]
+#0 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php(71): require()
+#1 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/LoadConfiguration.php(39): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->loadConfigurationFiles(Object(Illuminate\\Foundation\\Application), Object(Illuminate\\Config\\Repository))
+#2 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(213): Illuminate\\Foundation\\Bootstrap\\LoadConfiguration->bootstrap(Object(Illuminate\\Foundation\\Application))
+#3 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(162): Illuminate\\Foundation\\Application->bootstrapWith(Array)
+#4 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(146): Illuminate\\Foundation\\Http\\Kernel->bootstrap()
+#5 /var/www/seat/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(116): Illuminate\\Foundation\\Http\\Kernel->sendRequestThroughRouter(Object(Illuminate\\Http\\Request))
+#6 /var/www/seat/public/index.php(54): Illuminate\\Foundation\\Http\\Kernel->handle(Object(Illuminate\\Http\\Request))
+#7 {main}
+"}
+```
+
+# Checklist
+
+Check all boxes that apply to this issue :
+ - [ ] Bug description is provided
+ - [ ] Expected comportment is provided
+ - [ ] Request Path is provided
```

Comment:
all of the below checklist item should be lower case: 
f.e. `Request path` `Request method` etc.